### PR TITLE
Merge 19.5 code freeze

### PR DESCRIPTION
**Pull request by mokagio**

### CODE REVIEW

**Review by mokagio — COMMENTED** [review]
- [x] Switch to production Pods
- [x] New version header in `RELEASE-NOTE.txt` 
- [x] `Localizable.strings` updated 
- [x] `release_notes.txt` updated with notes from `RELEASE-NOTE.txt` for current version 
- [x] Version update in `.xcconfig` and `download_metadata.swift`

**Review by AliSoftware — COMMENTED** [review]
- [x] Pods updated to stable version
 - [x] New header in `RELEASE-NOTES.txt`
 - [x] JP and WP `release_notes.txt` updated with draft notes
 - [x] JP release notes audited for differences between WP and JP
    - ✅ Did not spot any item from the draft notes that didn't belong in JP, so all good to have the same notes for both in this release
 - [x] `WordPress/Resources/en.lproj/Localizable.strings` updated with latest strings
   - 🤔 ~I tried to do a diff on this file — locally, because GitHub apparently does not know how to diff UTF-16 files(†) — and saw no diff at all between this file in `release/19.5` and the one in `trunk` from last code-freeze.~
   - ~Which is intriguing because in the Release Notes I see in mentions of error messages being updated so I'd expect a change of string here… will investigate further.~
   - ☝️ nvm, see correction below; all is good.
 - [x] Version bump in `.xcconfig` files
   - [x] And in `fastlane/download_metadata.swift` (can't wait to get rid of that next one…)

_(†) Maybe we should update the `release-toolkit` to always generate those files using UTF-8 instead of UTF-16 btw, to make them more easily diff-able…_ 🤔

---

I especially wanted to look closer to the `Localizable.strings` file generated given that we recently switched to the new L10n tooling so wanted to be sure it worked as expected. I'll post this review now to highlight this note ASAP, and follow-up as I dig more into this to check if that empty diff is expected or not.

**Review by AliSoftware — APPROVED** [review]
> I'll post this review now to highlight this note ASAP, and follow-up as I dig more into this to check if that empty diff is expected or not.

Ok nvm, all my fault: my local `release/19.5` branch was not up-to-date when I tried to do that diff locally 🤦 

_Basically, uncaffeinated me accidentally created a local `release/19.5` branch from `trunk` this morning as I mistakenly started the code-freeze lane on WPiOS instead of WPAndroid… which means I had a `release/19.5` branch completely different from the real, remote `release/19.5` (and not tracking that remote branch)… no wonder I was comparing apples to oranges_ 🤦‍♂️

---

Diff for `Localizable.strings` file looks all good now that I've set things straight on my local machine after all 👍 :shipit: 
 - New strings like [this one](https://github.com/wordpress-mobile/WordPress-iOS/pull/17985/files#diff-bcd6d559bee73f68ff17c0635393e1fb2ea19083f00938971899f3fbb1fca254R404-R405) are in (pulled by `ios_generate_strings_files_from_code`)
 - New `/* Generated File. Do not edit */"` comment at top as well as the `/* MARK: <filename> */` sections introduced by the new `ios_merge_strings_files` action are indeed there  too 👍 
 - Copies like the ones for `NSPhotosLibraryUsageDescription` strings but which were using the English copy as key have been removed… (since obsolete `InfoPlistTranslator.m` file was removed)
 - … and reintroduced as `infoplist.NSPhotoLibrarUsageDescription` key from `en.lproj/InfoPlist.strings`, with the appropriate translator comment/hint (thanks to `ios_merge_strings_files` and use of key prefixes)

So TL;DR: everything worked exactly as expected after all 🎉 🍾

**Comment by AliSoftware** [comment]
@mokagio I see you did not initially request review from the Owl team on this PR, and didn't write a PR description either. I'm not sure if it was intentional — because you wanted to come back to this PR later after you ensured 19.5.0.0 would be approved by Apple in TestFlight before officially considering the PR ready and adding us as reviewers — or just an oversight.

Given that, I'll let you push the Merge button instead of merging the PR myself, just to be sure you didn't intend to revisit it before sending it for review.

**Comment by mokagio** [comment]
> So TL;DR: everything worked exactly as expected after all 🎉 🍾

🥂 

> I see you did not initially request review from the Owl team on this PR, and didn't write a PR description either.

The empty description was intentional, as I didn't have anything to add to the title 😅 But the missing review ping was an oversight after waiting for CI to give me the thumbs up. Thank you for asking 🙇‍♂️
